client/components/ProjectUserRequirements.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { AntDesign } from '@expo/vector-icons';
+import styled from 'styled-components/native';
+
+
+const ProjectUserRequirements = () => {
+
+    const UserView = styled.View`
+    display: flex;
+    flex-direction: row;
+    margin-top: 7px;
+    `
+
+    const UserText = styled.Text`
+    margin-left: 13px;
+    `
```

Comment:
styled components should be outside of the component definition as they need to be static